docs/src/versioning.md

```diff
@@ -21,7 +21,7 @@ The table below contains the *Quality-time* releases since the last minor of the
 
 | Version    | Date         | Mongo  | FC     | Migrations | Downgrade      | Upgrade         |
 |------------|--------------|--------|--------|------------|----------------|-----------------|
-| v5.14.0    | (unreleased) | v7     | v6     | added      | not possible   | n/a             |
+| v5.14.0    | (unreleased) | v7     | **v7** | added      | not possible   | n/a             |
```

Comment:
Shouldn't it be possible to downgrade?

Reply:
It's not the updated FC version that prevents the downgrade, but the added migrations.